docs/getting_started.rst

```diff
@@ -0,0 +1,208 @@
+Getting Started
+===============
+
+
+Who PSI/J is for
+----------------
+
+PSI/J is a library intended to make your job-launching code portable, or at
+least easier to port, across
+HPC centers. If you want your project to be able to request resources
+from one or more of Slurm, LSF, Flux, Cobalt, PBS, and your local machine,
+we think you will find that PSI/J simplifies your life considerably.
```

Comment:
Maybe "life" is a bit too broad :)

Reply:
How dare you understate the importance of PSI/J?